src/middlewared/middlewared/plugins/s3.py

```diff
@@ -72,6 +73,7 @@ async def config_extend(self, s3):
         ('edit', {'name': 'secret_key', 'method': lambda x: setattr(
             x, 'validators', [Match(r'^\w+$', explanation='Should only contain alphanumeric characters')]
         )}),
+        ('rm', {'name': 'tls_server_uri'}),
```

Comment:
I am not sure i follow this change, we should be able to change this and by doing this - we wouldn't be anymore ?

Reply:
It was not included in the entry schema so I decided it was not editable. Should it be?

Reply:
Yes, it definitely is editable :)